app/rifcs/rif_cs_generator.rb

```diff
@@ -98,6 +98,13 @@ def build_rif_cs
             end
           end
 
+          wrapper_object.contributors.each do |contributor|
+            xml.relatedObject do
+              xml.key contributor
+              xml.relation type: 'isOutputOf'
```

Comment:
As per the stakeholder's email, relation type should be ‘isEnrichedBy’ and description should be 'Contributor'.